### PR TITLE
COTECH-623 Update s2 with a page type for N&R

### PR DESCRIPTION
**Pull request by nandy-andy**
https://fandom.atlassian.net/browse/COTECH-623

### CODE REVIEW

File: @types/fandom/index.d.ts [hunk 1]
```diff
@@ -54,10 +54,16 @@ interface ITargetingParameters {
 	[key: string]: string | number;
 }
 
+interface ITargetingParameters {
```
Comment (pbartkowiak-fandom) [inline @types/fandom/index.d.ts:57]: The `ITargetingParameters` interface is defined 4 lines above your declaration. Did you declared it for the second time on purpose?

File: @types/fandom/index.d.ts [hunk 2]
```diff
@@ -54,10 +54,16 @@ interface ITargetingParameters {
...
+	playerName: string;
+}
+
 interface WindowFandomContext {
 	site: Partial<ISiteParameters>;
 	page: Partial<IPageParameters>;
 	tracking: Partial<ITrackingParameters>;
 	targeting: Partial<ITargetingParameters>;
 	partners: Partial<IPartnersParameters>;
+	video?: Partial<IVideoParameters>;
```
Comment (pbartkowiak-fandom) [inline @types/fandom/index.d.ts:68]: `IVideoParameters` interface declaration is missing. Did you mean `IVideoParameters` in line 57?

Comment (pbartkowiak-fandom) [inline @types/fandom/index.d.ts:68]: I don't think an interface declaration is a proper use case for the `Partial` utility type. The suggested use cases that I found are:
- When you need to only update some fields in an existing object
- When you need to pass constructor values to populate a new instance of the class

(source: https://timmousk.com/blog/typescript-partial/) 

Of course, the rule is not set in stone. Still, I would rather see explicite declarations:

```typescript
interface IVideoParameters {
	playerLoaded?: boolean;
	playerName?: string;
}

interface WindowFandomContext {
	...
	video: IVideoParameters;
}
```

WDYT?

File: @types/fandom/index.d.ts (hunk 1 above)
Comment (nandy-andy) [reply @types/fandom/index.d.ts:57]: Nope. Good catch, thanks! :)

File: @types/fandom/index.d.ts (hunk 2 above)
Comment (nandy-andy) [reply @types/fandom/index.d.ts:68]: Yep 🙈

Comment (nandy-andy) [reply @types/fandom/index.d.ts:68]: I have no strong feelings. I'm happy with your suggestion :) 👍 Thank you!

**Review by domjanik — APPROVED** [review]

**Review by pbartkowiak-fandom — APPROVED** [review]

**Review by dmnsobczak — COMMENTED** [review]
👍

**Pull request merged**